src/aae_util.erl

```diff
@@ -259,19 +312,6 @@ get_segmentid(B, K) ->
     Seg32 = leveled_tictac:keyto_segment32(make_binarykey(B, K)),
     leveled_tictac:get_segment(Seg32, ?TREE_SIZE).
 
-log_test() ->
-    log("D0001", [], []),
-    log_timer("G0001", [], os:timestamp(), []).
-
-log_warn_test() ->
-    ok = log("G0001", [], [], [warn, error]),
-    ok = log("G8888", [], [], [info, warn, error]),
-    SW = os:timestamp(),
-    ok = log_timer("G0001", [], SW, [], [warn, error]),
-    ok = log_timer("G8888", [], SW, [], [info, warn, error]),
```

Comment:
This test would fail if you had adapted it? Hinting that updating these simple tests may have a bit of value anyway